mlrun/projects/project.py

```diff
@@ -3390,6 +3390,28 @@ def set_model_monitoring_credentials(
                 "and set `rebuild_images=True`"
             )
 
+    def list_model_endpoints(
+        self,
+        model: Optional[str] = None,
+        function: Optional[str] = None,
+        labels: Optional[list[str]] = None,
+        start: str = "now-1h",
+        end: str = "now",
+        top_level: bool = False,
+        uids: Optional[list[str]] = None,
```

Comment:
Nice, though I'm not sure about the defaults of `start` and `end`.
Missing docstring